rcl/test/rcl/test_events.cpp

```diff
@@ -96,21 +99,12 @@ class CLASSNAME (TestEventFixture, RMW_IMPLEMENTATION) : public ::testing::Test
       &publisher_options);
   }
 
-  rcl_ret_t setup_subscriber(
-    const rmw_time_t & deadline,
-    const rmw_time_t & lifespan,
-    const rmw_time_t & liveliness_lease_duration,
-    const rmw_qos_liveliness_policy_t liveliness_policy)
+  rcl_ret_t setup_subscriber(const rmw_qos_profile_t qos_profile)
```

Comment:
Modified this function to take a `rmw_qos_profile_t` instead of each qos policy separately. This helps with resuse for the new test that is being added.